src/compiler/emitter.ts

```diff
@@ -2403,28 +2403,20 @@ module ts {
             }
 
             function emitMethod(node: MethodDeclaration) {
-                if (!isObjectLiteralMethod(node)) {
-                    return;
-                }

```

Comment:
To prevent class-methods from being emitted.  However, class methods never go through this code path, so it was never a problem.